plugins/modules/cloud/misc/proxmox.py

```diff
@@ -140,9 +140,10 @@
     type: str
   unprivileged:
     description:
-      - Indicate if the container should be unprivileged
+      - Indicate if the container should be unprivileged.
+      - The default value for this param is C(false) but that is being deprecated.
+      - It will be replaced with C(true) in community.general 7.0.0
```

Comment:
Nit picking (and largely a personal preference, more than a requirement): I think this would be better off as a single sentence:
```suggestion
      - >
        The default value for this param is C(false) but that is being deprecated
        and it will be replaced with C(true) in community.general 7.0.0.
```